packages/commonwealth/client/scripts/views/components/edit_profile.tsx

```diff
@@ -60,6 +60,8 @@ const EditProfileComponent = () => {
   const [addresses, setAddresses] = useState<AddressInfo[]>();
   const [displayNameValid, setDisplayNameValid] = useState(true);
   const [account, setAccount] = useState<Account>();
+  const [isUploadingProfileImage, setIsUploadingProfileImage] = useState(false);
```

Comment:
Is there a reason we don't use a more generic isUploading img here, how do we handle async uploads in a thread or comment for example? would love to see a consistent approach that gets us to not blocking the user as soon as possible (they should not have to wait for us to upload or process the video before they can save the form.)

Reply:
@mzparacha ^^

Reply:
ATM anywhere in the app where a media file is required to be uploaded, we block form submit actions. Ex: creating a post/comment, creating/updating a community, updating profile

Reply:
We can look into ways to not block users for these async actions, but I think that would be out of scope for this bug fix. Can create a spike tho if you'd like.